code/modules/food_and_drinks/machinery/oven.dm

```diff
@@ -243,6 +248,34 @@
 	max_items = 6
 	biggest_w_class = WEIGHT_CLASS_BULKY
 
+/obj/item/plate/oven_tray/attackby(obj/item/item, mob/living/user, params)
+
+	if(istype(item, /obj/item/storage))
```

Comment:
I think checking for `/obj/item/storage` directly isn't the best practice, rather you should check if the atom has an atom storage (ie, `.atom_storage`) and work off of that